src/lib/eZ/RichText/Converter/Xslt.php

```diff
@@ -77,7 +77,8 @@ protected function getXSLTProcessor()
             $hrefAttr = $xslDoc->createAttribute('href');
 
             // Prevents showing XSLTProcessor::importStylesheet() warning on Windows file system
-            $hrefAttr->value = str_replace('\\', '/', $stylesheet);
+            // & allows loading stylesheets if project's directory name contains space(s)
+            $hrefAttr->value = str_replace(['\\', ' '], ['/', '%20'], $stylesheet);
```

Comment:
TBH this is weird path sanitizing technique. I feel there are many more characters which might cause issues. Have you tried to apply `realpath` instead?

Reply:
or maybe `urlencode` ?

Reply:
Good catch guys! Changed it to use `realpath` as it seems to work fine.

@ezsystems/qa-team it **needs** to be tested on Windows machine